frontend/src/__tests__/cypress/cypress/pages/pipelinesList.ts

```diff
@@ -0,0 +1,19 @@
+class PipelinesList {
+  private testId = 'pipelines-list-page';
+
+  visit(projectName: string) {
+    cy.visitWithLogin(`/pipelines/${projectName}`);
+    this.wait();
+  }
+
+  wait() {
+    this.find();
+    cy.testA11y();
+  }
+
+  find() {
+    return cy.findByTestId(this.testId);
+  }
```

Comment:
`find` function name should be have a suffix describing what is being found. What are we finding? A general `find` is only useful when establishing a context where all subsequent selectors are chained from this. See `pages/component/Modal.ts` for example.

Is the `PipelinesSection` even used outside of project details? All selectors in a page object must be relevant to the page referenced by the `visit` function.

Reply:
At the time of writing my comment, the related `testId` referred to the `PipelinesSection` which is why I made that comment. If it's meant to target a list, likely the test ID was put in the wrong spot.

Reply:
Got it. I hoisted the test ID so I could get access to the project dropdown when navigating to the pipelines global page (I've been calling it the "last page" - but just the page we show when navigating to "Pipelines" from the left nav). As a result, I updated the page object name and test ID. I think the setup now makes sense, but please let me know if something is still off.